plugins/catalog-node/src/extensions.ts

```diff
@@ -31,6 +32,9 @@ export interface CatalogProcessingExtensionPoint {
     ...providers: Array<EntityProvider | Array<EntityProvider>>
   ): void;
   addPlaceholderResolver(key: string, resolver: PlaceholderResolver): void;
+  addLocationAnalyzers(
```

Comment:
I'd vote to not put this one in specifically the `CatalogProcessingExtensionPoint`. For one, it's not about processing. It's not always clear cut what "granularity" to make the extension points at, but this one was intentionally made to stay on the processor+provider work scope. And as a second aspect, the location analysis is arguably the less fleshed out feature that would benefit from some evolution, and its stability may be good to not lump together with the processing which is more mature. It's good to be able to break or evolve one, without affecting those who only care about the other.

So I'd vote for making a second extension point, something like an `catalogAnalysisExtensionPoint` next to the other one in catalog-node.